operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -339,4 +341,16 @@ public Counter getPeriodicReconciliationsCounter() {
     public AtomicInteger getResourceCounter() {
         return resourceCounter;
     }
+
+    public String getLoggingAppenders(String loggingConfiguration) {
+        OrderedProperties ops = new OrderedProperties();
+        ops.addStringPairs(loggingConfiguration);
+        StringBuilder result = new StringBuilder();
+        for (Map.Entry<String, String> entry: ops.asMap().entrySet()) {
+            if (entry.getKey().startsWith("log4j.appender")) {
+                result.append(entry.getKey()).append("=").append(entry.getValue());
+            }
+        }
+        return result.toString();
```

Comment:
This makes it sound like there are only appenders which need rolling update and loggers which can be set dynamically. Is there nothing else in the Log4j files? Would it be better to filter out here what you can set dynamically and keep the rest for the rolling update? I guess the same applies to #3469?

Reply:
You are right